tests/MongoDB.Driver.TestHelpers/DriverTestConfiguration.cs

```diff
@@ -134,7 +134,7 @@ public static DisposableMongoClient CreateDisposableClient(EventCapturer capture
             return CreateDisposableClient((ClusterBuilder c) => c.Subscribe(capturer));
         }
 
-        public static DisposableMongoClient CreateDisposableclient(MongoClientSettings settings)
+        public static DisposableMongoClient CreateDisposableClient(MongoClientSettings settings)
```

Comment:
Can be reverted. Originally I added one more method here and then noticed this typo. But now, since there is no other changes anymore, this can be reverted to have history a bit clear